docs/reference/building.md

```diff
@@ -9,7 +9,7 @@ Most people using QuPath won't need to build QuPath from source!
 Just download an existing installer from [qupath.github.io](https://qupath.github.io) and use that instead.
 :::
 
-## Command line
```

Comment:
Intended to disambiguate the "command line" section from "building from the command line"